eksctl/template.jsonnet

```diff
@@ -1,14 +1,29 @@
-// This file is a jinja2 template of a jsonnet template of a eksctl's cluster
-// configuration file, which is in turn can be used with the `eksctl` CLI to both
-// update and initialize a AWS EKS based cluster.
-//
-// This jinja2 template is only used by the deployer script as part of creating
-// new clusters. If a relevant change is made here or the dependent file
-// libsonnet/nodegroup.jsonnet, one may consider if we should manually update
-// already generated jsonnet files in this folder.
-//
-// Configuration reference: https://eksctl.io/usage/schema/
-//
+{#-
+    This file is a jinja2 template of a jsonnet template of a eksctl's cluster
+    configuration file, which in turn is to be used with the eksctl CLI to both
+    update and initialize an AWS EKS based cluster.
+
+    This jinja2 template is used by the deployer script's generate-aws-cluster
+    command as part of creating new clusters.
+
+    References:
+    - https://infrastructure.2i2c.org/en/latest/hub-deployment-guide/new-cluster/aws.html#generate-cluster-files
+-#}
+/*
+    This file is a jsonnet template of a eksctl's cluster configuration file,
+    that is used with the eksctl CLI to both update and initialize an AWS EKS
+    based cluster.
+
+    This file has in turn been generated from eksctl/template.jsonnet which is
+    relevant to compare with for changes over time.
+
+    To use jsonnet to generate an eksctl configuration file from this, do:
+
+        jsonnet << cluster_name >>.jsonnet > eksctl-config.yaml
+
+    References:
+    - https://eksctl.io/usage/schema/
+*/
```

Comment:
This is a jsonnet comment to be retained after being generated from the jinja2 template, while the `{#` parts above are jinja2 comments just for this template.jsonnet file.